python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -310,7 +312,7 @@ def adjust_literal_dtype(literal: expr.Literal) -> expr.Literal:  # pragma: no c
             )
         return literal
 
-    def maybe_adjust_binop(e) -> expr.Expr:  # pragma: no cover
+    def maybe_adjust_binop(e: expr.NamedExpr) -> expr.NamedExpr:  # pragma: no cover
```

Comment:
I *think* that the previous return type was incorrect. Since the input `e` was `Any` (by default) returning `e` unmodified type-checked since `Any` is always OK.